indico/web/templates/forms/multiple_tag_select_widget.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'forms/base_widget.html' %}
+
+{% block html %}
+    <div id="{{ field.id }}-wrapper"></div>
+{% endblock %}
+
+{% block javascript %}
+    <script>
+        setupMultipleTagSelectWidget({
+            fieldId: {{ field.id | tojson }},
+            wrapperId: "{{ field.id }}-wrapper",
```

Comment:
```suggestion
            wrapperId: '{{ field.id }}-wrapper',
```

Reply:
But I'd just derive this from `fieldId` in the code and not pass it separately...